package/linux/openpilot.desktop

```diff
@@ -1,11 +1,11 @@
 [Desktop Entry]
 Version=0.1.0
 Encoding=UTF-8
-Name=OpenPilot GCS
-Exec=openpilotgcs
-TryExec=openpilotgcs
-Comment=Configure, Tune, Diagnose, Track, & Upgrade FW for OpenPilot solutions
+Name=LibrePilot GCS
+Exec=librepilot-gcs
+TryExec=librepilot-gcs
```

Comment:
Should we remove the hard coded name from here and then fix it via sed and defines in the Makefile?